src/main/java/gov/nasa/pds/tools/validate/BundleManager.java

```diff
@@ -448,18 +466,22 @@ public static Target getLatestBundle() {
         try {
             // Get the parent directory of url and crawl for files that starts with 'bundle'
             String dirName     = (new File(url.getPath())).getParent();
-            LOG.debug("findOtherBundleFiles:dirName {}",dirName);
+            LOG.debug("findOtherCollectionFiles:dirName {}",dirName);
             Crawler crawler = CrawlerFactory.newInstance(new File(dirName).toURI().toURL());
             IOFileFilter regexFileFilter = new RegexFileFilter(COLLECTION_LABEL_PATTERN);
-            LOG.debug("findOtherBundleFiles:crawler {}",crawler);
-            allFiles = crawler.crawl(new File(dirName).toURI().toURL(),regexFileFilter);
+            LOG.debug("findOtherCollectionFiles:crawler {}",crawler);
+            //allFiles = crawler.crawl(new File(dirName).toURI().toURL(),regexFileFilter);
+            // Note: For some strange reason, the crawler goes into an infinite loop using the above call
+            //       so we will use an alternate call to get the list of files.
+            LABEL_EXTENSIONS_LIST[0] = LABEL_EXTENSION;
```

Comment:
@qchaupds similar to my other comment, this doesn't really seem like good programming practice? I feel like the array should be set as a constant when the constant is declared. not the end of the world, but just something to keep in mind moving forward.